src/index.css

```diff
@@ -1,6 +1,6 @@
 body {
   margin: 10px;
   padding: 0;
-  width: 750px;
+  max-width: 750px;
```

Comment:
Yeah I made that change into master already. It shouldn't be a problem at merging.